docs/workflows/genomic_characterization/theiameta.md

```diff
@@ -324,6 +324,11 @@ The TheiaMeta_Illumina_PE workflow processes Illumina paired-end (PE) reads ge
 | largest_contig | Int | Largest contig size |
 | metaspades_docker | String | Docker image of metaspades |
 | metaspades_version | String | Version of metaspades |
+| midas_primary_genus | String | Primary genus detected by MIDAS |
+| midas_report | File | MIDAS report file |
+| midas_secondary_genus | String | Secondary genus detected by MIDAS |
+| midas_secondary_genus_abundance | Float | Abundance of the secondary genus |
+| midas_secondary_genus_coverage | Float | Coverage of the secondary genus |
```

Comment:
@fraser-combe some output files are missing from the table, namely the midas_report which is the most important output from a metagenomics perspective

Reply:
Do you mean the terra output table, you can see the midas_report tsv file as output in example in he PR when midas = true, maybe I am misreading your comment?
![image](https://github.com/user-attachments/assets/796797a7-b645-4480-9cf5-2dec52387250)